pages/common/vela.md

```diff
@@ -0,0 +1,8 @@
+# vela
+
+> Command line tools for vela pipeline.
+> More information: <https://go-vela.github.io/docs/>.
+
+- Trigger pipeline to run from git tagged version:
+
+`vela add deployment --org {{git org}} --repo {{repo name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git tag}} --description "{{Deploy description}}"`
```

Comment:
We use `snake_case` for variables:
```suggestion
`vela add deployment --org {{organization}} --repo {{repository_name}} --target {{dev/stage/prod/other}} --ref refs/tags/{{git_tag}} --description "{{deploy_description}}"`
```
The `--org` which values could have? `github`?
What is a `git org`?